Source/JavaScriptCore/runtime/ClonedArguments.cpp

```diff
@@ -150,7 +154,7 @@ ClonedArguments* ClonedArguments::createWithMachineFrame(JSGlobalObject* globalO
 
 ClonedArguments* ClonedArguments::createByCopyingFrom(JSGlobalObject* globalObject, Structure* structure, Register* argumentStart, unsigned length, JSFunction* callee, Butterfly* butterfly)
 {
-    ClonedArguments* result = createEmpty(globalObject, structure, callee, length, butterfly);
+    ClonedArguments* result = createEmpty(globalObject->vm(), globalObject, structure, callee, length, butterfly);
```

Comment:
Why we want to pass the `VM` since we can directly get it from the passed `globalObject` in `createEmpty`?

Reply:
Ok, I see.
```
ClonedArguments::createEmpty(vm, nullptr, globalObject->clonedArgumentsStructure(), callee, length, nullptr);
```